sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -345,52 +349,33 @@ public void build(AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder) {
             ));
     }
 
-    private void internalBuildAndFreeze() {
+    // One caveat: ConfigProperties will get used only once when initializing/starting StatsbeatModule.
+    // When a customer call build(AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder) multiple times with a diff ConfigProperties each time,
+    // the new ConfigProperties will not get applied to StatsbeatModule because of "frozen" guard. Luckily, we're using it for testing only.
+    // We might need to revisit this approach later.
```

Comment:
```suggestion
    // the new ConfigProperties will not get applied to StatsbeatModule because of "frozen" guard. Luckily, we're using the config properties in StatsbeatModule for testing only.
    // We might need to revisit this approach later.
```